sacred/experiment.py

```diff
@@ -39,7 +40,8 @@ class Experiment(Ingredient):
     def __init__(self, name: Optional[str] = None,
                  ingredients: Sequence[Ingredient] = (),
                  interactive: bool = False,
-                 base_dir: Optional[PathType] = None):
+                 base_dir: Optional[PathType] = None,
+                 additional_gatherers: Dict[str, Callable] = None):
```

Comment:
```suggestion
                 additional_gatherers: Dict[str, Callable[[]], str] = None):
```
We can explicitly type that this should be a function that does not need any arguments. I also think that limiting the return type to str could make sense, but probably everything that can be serialized is fine.

Reply:
I've put `Any` as return type. We don't want tools to throw errors when the code is correct. It's better to allow wrong code instead.